extension/shops/shop.mjs

```diff
@@ -100,14 +100,14 @@ export class StatefulShop extends Shop {
 
   async scheduleRendering({ render, cleanup, fetchData }) {
     new MutationObserver(async mutations => {
+      if (this.shouldCleanup(mutations)) cleanup();
```

Comment:
I've checked all shops using StatefulShop (electroworld, rohlik, itesco, kosik) and all seem fine with this change